### DISCUSSION TITLE
Feature Request to move the Ubuntu Image to Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by jhawkins1** [grafana/grafana#59040] [General]
There are a few old security vulnerabilities in 20.04 that Ubuntu/Canonical does not intend on fixing, but are fixed in 22.04.  This request is for the Grafana Project to bump Ubuntu Container Image to use 22.04.

**Comment by adeptg** [comment]
I've created an MR https://github.com/grafana/grafana/pull/60668

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
